### PR TITLE
BZ1923767: adding module to disable OperatorHub in restricted installs

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1923767

4.6+

Sample preview: https://deploy-preview-33462--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html

### CODE REVIEW

File: modules/olm-restricted-networks-configuring-operatorhub.adoc [hunk 1]
```diff
@@ -14,8 +29,11 @@ endif::[]
 = Disabling the default OperatorHub sources
 
 Operator catalogs that source content provided by Red Hat and community projects are configured for OperatorHub by default during an {product-title} installation.
+ifndef::olm-managing-custom-catalogs[]
```
Comment (kalexand-rh) [inline modules/olm-restricted-networks-configuring-operatorhub.adoc:32]: @adellape, are you ok with this if statement?

**Comment by kalexand-rh** [comment]
@dmesser, @SNiemann15, will you PTAL?

**Comment by SNiemann15** [comment]
@kalexand-rh For Power I have the ack from QE, for Z I will let you know once I get it.

**Comment by kalexand-rh** [comment]
@maxwelldb, @rolfedh, any thoughts on how this rolls up in restricted network [OpenStack](https://deploy-preview-33462--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-restricted.html#olm-restricted-networks-operatorhub_installing-openstack-installer-restricted) and [RHV](https://deploy-preview-33462--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-restricted-network.html#olm-restricted-networks-operatorhub_installing-rhv-restricted-network)?

**Comment by rolfedh** [comment]
> @maxwelldb, @rolfedh, any thoughts on how this rolls up in restricted network [OpenStack](https://deploy-preview-33462--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-restricted.html#olm-restricted-networks-operatorhub_installing-openstack-installer-restricted) and [RHV](https://deploy-preview-33462--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-restricted-network.html#olm-restricted-networks-operatorhub_installing-rhv-restricted-network)?

Sorry, @kalexand-rh, I've been away from RHV for too long now. @emarcusRH @stoobie, would you help Kathryn with this question?

**Comment by SNiemann15** [comment]
> @kalexand-rh For Power I have the ack from QE, for Z I will let you know once I get it.

@kalexand-rh The Z folks have also given their ok.

**Review by dmesser — APPROVED** [review]

**Comment by kalexand-rh** [comment]
@rlobillo or @duanwei33, can one of you approve this addition from purely an [OpenStack](https://deploy-preview-33462--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-restricted.html#olm-restricted-networks-operatorhub_installing-openstack-installer-restricted) perspective?

**Comment by jianzhangbjz** [comment]
I'm not sure which/how many platforms need to add this doc, but for disabling OperatorHub in restricted cluster, it looks good. 
/lgtm

**Comment by duanwei33** [comment]
@kalexand-rh Sorry I'm afraid I'm not the right person for this review.

**Comment by kalexand-rh** [comment]
> @kalexand-rh Sorry I'm afraid I'm not the right person for this review.
@duanwei33, do you know who might be able to give an opinion on the ShiftStack side?

**Review by mikemckiernan — APPROVED** [review]
Bummer for me.  I was hoping that you typed and would be willing to revise the text "...that source content provided..."  I found that "source" as a verb was a little tricky _for me_ to noodle through.

Looks good.

**Comment by kalexand-rh** [comment]
cherrypick enterprise-4.7

**Comment by kalexand-rh** [comment]
Per @sjstout, I am merging this one.

**Pull request merged**